Makefile

```diff
@@ -32,6 +32,7 @@ SANDBOX_NAMESPACE ?= sandbox-rhoam-operator
 # These tags are modified by the prepare-release script.
 RHMI_TAG ?= 2.9.0
 RHOAM_TAG ?= 1.22.0
+MULTITENANT_RHOAM_TAG ?= 1.21.0
```

Comment:
@carlkyrillos Similar to Michals previous comment, I'm not sure there is a need for a different version tag? 
I think maybe the issue is that we should be building 1.21.0 based on the 1.21.0 code base and not the 1.22.0 code base. So, it's more of a process issue.

Reply:
After discussing this on the main channel, we decided to keep this approach with separate versions for MT and standard RHOAM in order to simplify support for patch releases.